.github/release-drafter.yml

```diff
@@ -0,0 +1,50 @@
+categories:
+  - title: '🚀 Features'
+    labels:
+      - 'kind/enhancement'
+  - title: '🐛 Bug Fixes'
+    labels:
+      - 'kind/bug'
+  - title: '🧰 Maintenance'
+    labels:
+      - 'kind/chore'
+      - 'area/dependencies'
+
+exclude-labels:
+  - duplicate
+  - invalid
+  - later
+  - wontfix
+  - question
+  - skip-changelog
+
+change-template: '- $TITLE (#$NUMBER)'
+change-title-escapes: '\<*_&' # You can add # and @ to disable mentions, and add ` to disable code blocks.
+tag-template: 'v$RESOLVED_VERSION'
```

Comment:
Looks good!

I have to say I am uncomfortable with setting a version programmatically already.
I would like first to have a linter so all PRs follow the conventional commits style, so we are sure to never forget a possible breaking change.

Even with that, I would be more comfortable producing some releases via release-drafter yet keeping tags and version bumps manually.

Reply:
I've changed the configuration to keep the next release with the "Draft" name. The release workflow will update the release name with the tag pushed by the maintainers.